runtime/compiler/env/j9method.cpp

```diff
@@ -1637,6 +1661,9 @@ TR_ResolvedRelocatableJ9Method::getResolvedPossiblyPrivateVirtualMethod(
          ignoreRtResolve,
          unresolvedInCP);
 
+   if (comp->getOption(TR_UseSymbolValidationManager))
```

Comment:
Because the validation was added in `TR_ResolvedJ9Method::getResolvedPossiblyPrivateVirtualMethod`. It had to be put in there because the validation had to be added before the call `resolvedMethod = createResolvedMethodFromJ9Method(comp, cpIndex, vTableOffset, ramMethod, unresolvedInCP, aotStats);` which will result in an aborted compilation if, in the constructor of `TR_ResolvedRelocatableJ9Method`, the j9method wasn't already validated. Probably some opportunity for cleanup.